test/models/trayecctoy_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class TrayecctoyTest < ActiveSupport::TestCase
```

Comment:
which one is the correct TrayecctoyTest or TrajectoryTest?

Reply:
ya its delete trayecctoy